gulpfile.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var webpack = require('gulp-webpack');
+var minifyCss = require('gulp-minify-css');
+var uglify = require('gulp-uglify');
+
+gulp.task('sass', function () {
+  gulp.src('./sass/**/*.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('../dev/css/'));
+});
+
+gulp.task('sass:watch', function () {
+  gulp.watch('./sass/**/*.scss', ['sass']);
+});
+
+gulp.task('webpackdev', function() {
+  return gulp.src('./dev/js/main.js')
+    .pipe(webpack({
+      output: {
+        filename: 'bundle.js'
+      }
+    }))
+    .pipe(uglify())
+    .pipe(gulp.dest('./build/'));
+});
+
+gulp.task('minify-css', function() {
+  return gulp.src('./dev/CSS/*.css')
+    .pipe(minifyCss({compatibility: 'ie8'}))
+    .pipe(gulp.dest('./build/CSS/'));
+});
+
+gulp.task('compress', function() {
+  return gulp.src('lib/*.js')
+    .pipe(uglify())
+    .pipe(gulp.dest('build'));
+});
+
+gulp.task('copy', function() {
+  var opts = {
+    conditionals: true,
+    spare:true
+  };
+    return gulp.src('dev/**/*.html')
+    .pipe(gulp.dest('build/'))
+    //.pipe(minifyHTML(opts))
+    .pipe(gulp.dest('./build/'));
+});
+
+gulp.task('build', ['copy', 'copyServer', 'webpackdev', 'minify-css']);
+gulp.task('default', ['build']);
```

Comment:
I don't see copyServer, did that task get deleted? This will probably throw an error in that case.